src/components/ArrowKeyFocusManager.js

```diff
@@ -0,0 +1,139 @@
+import _ from 'underscore';
+import {Component} from 'react';
+import PropTypes from 'prop-types';
+import * as PropTypeUtils from '../libs/PropTypeUtils';
+import CONST from '../CONST';
+import KeyboardShortcut from '../libs/KeyboardShortcut';
+
+const propTypes = {
+    /** Children to render. */
+    children: PropTypes.oneOfType([
+        PropTypes.func,
+        PropTypes.node,
+    ]).isRequired,
+
+    /** The initial focused index. */
+    initialFocusedIndex: PropTypeUtils.integerPropType,
+
+    /** The number of items in the list that this component is managing focus for. */
+    listLength: PropTypeUtils.wholeNumberPropType,
+
+    /** A callback executed when the focused input changes. */
+    onFocusedIndexChanged: PropTypes.func,
+
+    /** A callback executed when the enter key is pressed. */
+    onEnterKeyPressed: PropTypes.func,
+
+    /** Should the enter key event bubble? */
+    shouldEnterKeyEventBubble: PropTypes.oneOfType([PropTypes.bool, PropTypes.func]),
+};
+
+const defaultProps = {
+    initialFocusedIndex: 0,
+    listLength: 0,
+    onFocusedIndexChanged: () => {},
+    onEnterKeyPressed: undefined,
```

Comment:
should this be an empty function as well since it's defined as `onEnterKeyPressed: PropTypes.func,`?

```suggestion
    onEnterKeyPressed: () => {},
```